### ISSUE TITLE
use_dir_size broken?

### ISSUE THREAD

**Issue by bluboyo** [glftpd/glftpd#72]
use_dir_size broken? if i set it to m or g i get kb. If i set it to k i get b. 

ex. "use_dir_size g /site/TV"

OS Debian 11.6
gl 2.13

**Comment by glftpd** [comment]
seems to work fine for me ? maybe some special conditions ?

**Comment by bluboyo** [comment]
So you see the total size of the folder?

**Comment by glftpd** [comment]
yeah, if you find some more details please let me know. A special filesystem maybe ?

**Comment by bluboyo** [comment]
Its a mounted folder in a container. Filesystem is ZFS.

**Comment by glftpd** [comment]
any chance you can verify it works fine for you on a non-mounted fs ?

**Comment by bluboyo** [comment]
Tested and its not working. Lets say the content is 10gib. Then its shows maybe 3-20b.

**Comment by glftpd** [comment]
Does it only not work with ZFS? Maybe ZFS isn't returning the correct data to glftpd. What happens if you do normal "ls" in the shell ?
